lib/teen_process.js

```diff
@@ -100,6 +100,11 @@ class SubProcess extends EventEmitter {
     this.opts = opts;
   }
 
+  get isRunning () {
+    // presence of `proc` means we have connected and started
+    return !!this.proc;
```

Comment:
I guess this is a question: do we want to return `isRunning` only when `proc` exists or when it exists and the user-defined startDetector has also been validated?

If we mean "running" to just mean "there's a subprocess happening" then this is right. And that's probably OK.